pkgs/applications/editors/emacs/elisp-packages/manual-packages/zstd/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  melpaBuild,
+  pkgs,
```

Comment:
It is better to list `stdenv` and `zstd` directly because it is more clear and I guess it is better for cross compilation.

To avoid infinite recursion, do something like this: https://github.com/NixOS/nixpkgs/blob/2a3a53a2fe8ae563c3c3c4d39c189ebc31acc308/pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix#L28.